src/convnets/inception.jl

```diff
@@ -176,21 +178,392 @@ See also [`inception3`](#).
 
 !!! warning
     
-    `Inception3` does not currently support pretrained weights.
+    `Inceptionv3` does not currently support pretrained weights.
+"""
+struct Inceptionv3
+    layers::Any
+end
+
+function Inceptionv3(; pretrain = false, nclasses = 1000)
+    layers = inceptionv3(; nclasses = nclasses)
+    pretrain && loadpretrain!(layers, "Inceptionv3")
+    return Inceptionv3(layers)
+end
+
+@functor Inceptionv3
+
+(m::Inceptionv3)(x) = m.layers(x)
+
+backbone(m::Inceptionv3) = m.layers[1]
+classifier(m::Inceptionv3) = m.layers[2]
+
+@deprecate Inception3 Inceptionv3
+@deprecate inception3 inceptionv3
```

Comment:
Let's do deprecations in a separate "deprecations.jl" file at the source root. We don't have many, but it makes it easier to check what needs to get dropped on a breaking version.